### PR TITLE
[EDU-998] - Add tools page

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Add a tools page to expose various developer tools and projects.

* [Epic EDU-996](https://ably.atlassian.net/browse/EDU-996)
* [EDU-998](https://ably.atlassian.net/browse/EDU-998)

## Review

* [Tools page](https://ably-docs-edu-998-tools-sp4rq1.herokuapp.com/tools/)

### CODE REVIEW

File: content/root/tools.textile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  <img src="/images/screenshots/developer-console.png" style="width: 100%" alt="Developer Console">
+</a>
+
+You can also attach to a specific channel to view messages. You can also publish messages directly in the console:
+
+<a href="/images/screenshots/developer-console-channels.png" target="_blank">
+  <img src="/images/screenshots/developer-console-channels.png" style="width: 100%" alt="Developer Console - Channels">
+</a>
+
+The developer console also provides you with some REST API Curl command snippets that can help you test out Ably quickly:
```
Comment (leggetter) [inline content/root/tools.textile:59]: Worth emphasising that the credentials are pre-filled which is a big part of making them helpful?

File: content/root/tools.textile [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+A GitHub Action to use the "Ably Control API":https://ably.com/docs/control-api. You can use this action to:
+
+* Create an Ably application.
+* Add an API key to an application, with a list of capabilities.
+
+"Find it here.":https://github.com/ably-labs/ably-control-api-action
+
+h2(#developer-console). Developer console
+
+The "Ably dashboard":ably.com/dashboard contains a developer console. In the developer console you can view connection events. The following screenshot ilustrates an example connection:
```
Comment (Sapples) [inline content/root/tools.textile:48]: *illustrates

File: content/root/tools.textile [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Use Ably in your React application using idiomatic React Hooks.
+
+Using this package you can:
+
+* Interact with Ably channels using a React hook.
+* Send messages via Ably using the channel instances the hooks provide.
+* Get notifications of user presence on channels.
+* Send presence updates.
+
+The hooks provide a simplified syntax for interacting with Ably, and manage the lifecycle of the Ably SDK instances for you taking care to subscribe and unsubscribe to channels and events when your React components re-render.
```
Comment (Sapples) [inline content/root/tools.textile:81]: First part doesn't flow quite right, one of these feels better to me:

`The hooks provide a simplified syntax for interacting with Ably, and **managing** the lifecycle`
Or
`The hooks provide a simplified syntax **to interact** with Ably, and manage the lifecycle`

**Review by Sapples — CHANGES_REQUESTED** [review]
Couple of small comments.

File: content/root/tools.textile (hunk 2 above)
Comment (tbedford) [reply content/root/tools.textile:48]: Well spotted!

File: content/root/tools.textile (hunk 3 above)
Comment (tbedford) [reply content/root/tools.textile:81]: Rewritten!

**Review by Sapples — APPROVED** [review]
LGTM

**Pull request merged**